Quantum-Control-Applications/Superconducting/Single Fixed Transmon/time_of_flight.py

```diff
@@ -27,26 +28,36 @@
 #####################################
 #  Open Communication with the QOP  #
 #####################################
-qmm = QuantumMachinesManager(host=qop_ip, port=qop_port)
-
-qm = qmm.open_qm(config)
-job = qm.execute(tof_cal)
-res_handles = job.result_handles
-res_handles.wait_for_all_values()
-adc1 = res_handles.get("adc1").fetch_all() / 2**12
-adc2 = res_handles.get("adc2").fetch_all() / 2**12
-adc1_single_run = res_handles.get("adc1_single_run").fetch_all() / 2**12
-adc2_single_run = res_handles.get("adc2_single_run").fetch_all() / 2**12
-
-plt.figure()
-plt.title("Single run (Check ADCs saturation)")
-plt.plot(adc1_single_run)
-plt.plot(adc2_single_run)
-plt.show()
-
-plt.figure()
-plt.title("Averaged run (Check ToF & DC Offset)")
-plt.plot(adc1)
-plt.plot(adc2)
-plt.show()
-print(f"Input1 mean: {np.mean(adc1)} V\n" f"Input2 mean: {np.mean(adc2)} V")
+# qmm = QuantumMachinesManager(host=qop_ip, port=qop_port)
+qmm = QuantumMachinesManager()
```

Comment:
I believe we still need the IP address here

Reply:
yes, this was my mistake